spectator-reg-atlas/src/main/java/com/netflix/spectator/atlas/impl/DefaultPublisher.java

```diff
@@ -46,7 +45,7 @@ public final class DefaultPublisher implements Publisher {
 
   private static final String CLOCK_SKEW_TIMER = "spectator.atlas.clockSkew";
 
-  private final Logger logger = LoggerFactory.getLogger(AtlasRegistry.class);
+  private static final Logger LOGGER = LoggerFactory.getLogger(DefaultPublisher.class);
```

Comment:
Changing to static is fine, but the logger name shouldn't be changed at this time. There are some use-cases that expect these logs to come from the registry class.

Reply:
I get that, my reasoning was that it can be confusing if the log messages aren't tied to the actual location where they're being logged, but I guess it's too late now to fix it if someone is depending on it.

Reply:
Okay, I've updated the PR to just make the field static / final instead.

Reply:
> my reasoning was that it can be confusing if the log messages aren't tied to the actual location where they're being logged

Since logs are queryable and can be used for alerts, we try to be careful with certain messages, in particular the notifications round validation errors as many teams look at those.